src/archanaNJun20/ExtractItemId.java

```diff
@@ -0,0 +1,29 @@
+package archanaNJun20;
+
+public class ExtractItemId {
+	void printItemId(String input) {
+		String output = "";
+		if (input.contains("itemId")) {
+			String temp = input.substring(input.indexOf("12345"));
```

Comment:
i don't think that we can directly provide the value "12345" and extract the same value in the program.
You can substring the input till "itemId=" and than extract the Id i.e. "12345"

Reply:
The code is updated.